backend/src/test/java/gov/cdc/usds/simplereport/test_util/TestDataFactory.java

```diff
@@ -428,13 +428,18 @@ public TestEvent createTestEvent(Person p, Facility f) {
   public TestEvent createTestEvent(Person p, Facility f, AskOnEntrySurvey s, TestResult r, Date d) {
     TestOrder o = createTestOrder(p, f, s);
     o.setDateTestedBackdate(d);
-    Result result = new Result(o, diseaseService.covid(), r);
-    TestEvent e = testEventRepository.save(new TestEvent(o, false, Set.of(result)));
+    Result orderResult = new Result(o, diseaseService.covid(), r);
+
+    TestEvent e = new TestEvent(o, false);
+    testEventRepository.save(e);
+
+    Result copiedResult = new Result(orderResult);
+    copiedResult.setTestEvent(e);
+    resultRepository.save(copiedResult);
```

Comment:
do we need to save the `orderResult` in the repo and add it to the order as well? otherwise do we need to create it?